Source/WebCore/html/HTMLAttachmentElement.cpp

```diff
@@ -70,14 +75,122 @@ HTMLAttachmentElement::~HTMLAttachmentElement() = default;
 
 Ref<HTMLAttachmentElement> HTMLAttachmentElement::create(const QualifiedName& tagName, Document& document)
 {
-    return adoptRef(*new HTMLAttachmentElement(tagName, document));
+    Ref<HTMLAttachmentElement> attachment = adoptRef(*new HTMLAttachmentElement(tagName, document));
+    if (DeprecatedGlobalSettings::attachmentWideLayoutEnabled() && DeprecatedGlobalSettings::attachmentElementEnabled()) {
```

Comment:
```suggestion
    if (document.settings().attachmentWideLayoutEnabled() && DeprecatedGlobalSettings::attachmentElementEnabled()) {
```

To avoid adding a new DeprecatedGlobalSetting.

Reply:
I used a DeprecatedGlobalSetting to be consistent with the existing `attachmentElementEnabled` and keep them together. But I'll have a look...